### DISCUSSION TITLE
WAS's Comprehensive Node Suite

### DISCUSSION THREAD

**Discussion by WASasquatch** [Comfy-Org/ComfyUI#241] [Show and tell]
I have released a new all-in-one Node Suite with tons of goodies and moe on the way.

CivitAI: https://civitai.com/models/20793/wass-comprehensive-node-suite-comfyui
Github: https://github.com/WASasquatch/was-node-suite-comfyui

**Comment by GrimblyGorn** [comment]
Even packed all neatly into one file. Very nice and neat. Thanks :)

**Reply by GrimblyGorn** [reply]
Line 1546 needed a closing bracket for it to work though.
````
self.output_dir = os.path.normpath(output_path.strip())
````

**Reply by WASasquatch** [reply]
Yeah, that's what I get for leaving the window open and coming back, must have deleted a character cause that was literally one of the last things tested.
